core/src/Microsoft.Identity.Core.csproj

```diff
@@ -127,4 +127,6 @@
       <Version>27.0.2</Version>
     </PackageReference>
   </ItemGroup>
+	  <Target Name="ProjectReferenceSolution" BeforeTargets="ResolveAssemblyReferences" DependsOnTargets="ResolveProjectReferences">
```

Comment:
What is the purpose of this?

Reply:
This solves one of the visual studio bugs that results in broken project references in multi-targeted solutions. I only put it into test projects as they are the only ones effected.

Reply:
#Resolved